backend/src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -259,20 +270,14 @@
     @Produces(APPLICATION_JSON)
     @Operation(summary = "Retrieve the behavior groups of a bundle.")
     @RolesAllowed(RbacIdentityProvider.RBAC_READ_NOTIFICATIONS)
-    public Uni<List<BehaviorGroup>> findBehaviorGroupsByBundleId(@Context SecurityContext sec, @PathParam("bundleId") UUID bundleId) {
+    public Uni<List<BehaviorGroup>> findBehaviorGroupsByBundleId(@Context SecurityContext sec,
+            @PathParam("bundleId") UUID bundleId) {
         return sessionFactory.withSession(session -> {
-            return getAccountId(sec)
-                    .onItem().transformToUni(accountId -> behaviorGroupResources.findByBundleId(accountId, bundleId))
-                    .onItem().call(behaviorGroups -> endpointResources.loadProperties(
-                                    behaviorGroups
-                                            .stream()
-                                            .map(BehaviorGroup::getActions)
-                                            .filter(Objects::nonNull)
-                                            .flatMap(Collection::stream)
-                                            .map(BehaviorGroupAction::getEndpoint)
-                                            .collect(Collectors.toList())
-                            )
-                    );
+            return getAccountId(sec).onItem()
+                    .transformToUni(accountId -> behaviorGroupResources.findByBundleId(accountId, bundleId)).onItem()
+                    .call(behaviorGroups -> endpointResources.loadProperties(behaviorGroups.stream()
```

Comment:
Here's an example where the reactive code is harder to read after the indentation in enforced.

`.onItem().transformToUni(...)` and `.onItem().call(...)` are kind of a single reactive instruction and should be kept together for the sake of readability.